stories/sample-pages/layout-1.stories.jsx

```diff
@@ -3,6 +3,9 @@ import Button from '@leafygreen-ui/button';
 
 export default {
   title: 'Sample Pages (WIP)/Layout 1',
+  parameters: {
+    chromatic: { disableSnapshot: true },
```

Comment:
ditto here

Reply:
We only use component stories on the website.

Reply:
Just Banner - I was saying we could rm these two files